mmv1/products/certificatemanager/api.yaml

```diff
@@ -212,4 +212,207 @@ objects:
+                An IP configuration where this Certificate Map is serving
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'ipAddress'
+                    description : |
+                      An external IP address
+                  - !ruby/object:Api::Type::Array
+                    name: 'ports'
+                    description : |
+                      A list of ports
+                    item_type: Api::Type::Integer
+            - !ruby/object:Api::Type::String
+              name: 'targetHttpsProxy'
+              description: |
+                Proxy name must be in the format projects/*/locations/*/targetHttpsProxies/*.
+                This field is part of a union field `target_proxy`: Only one of `targetHttpsProxy` or
+                `targetSslProxy` may be set.
+            - !ruby/object:Api::Type::String
+              name: 'targetSslProxy'
+              description: |
+                Proxy name must be in the format projects/*/locations/*/targetSslProxies/*.
+                This field is part of a union field `target_proxy`: Only one of `targetHttpsProxy` or
+                `targetSslProxy` may be set.
+  - !ruby/object:Api::Resource
+      name: 'CertificateMapEntry'
+      base_url: 'projects/{{project}}/locations/global/certificateMaps'
+      create_url: 'projects/{{project}}/locations/global/certificateMaps?certificateMapId={{name}}'
+      self_link: 'projects/{{project}}/locations/global/certificateMaps/{{name}}'
+      update_verb: :PATCH
+      update_mask: true
+      description: |
+        CertificateMapEntry is a list of certificate configurations,
+        that have been issued for a particular hostname 
+      async: !ruby/object:Api::OpAsync
+        operation: !ruby/object:Api::OpAsync::Operation
+          path: 'name'
+          base_url: '{{op_id}}'
+          wait_ms: 1000
+        result: !ruby/object:Api::OpAsync::Result
+          path: 'response'
+        status: !ruby/object:Api::OpAsync::Status
+          path: 'done'
+          complete: true
+          allowed:
+            - true
+            - false
+        error: !ruby/object:Api::OpAsync::Error
+          path: 'error'
+          message: 'message'
+      parameters:
+        - !ruby/object:Api::Type::String
+          name: 'name'
+          required: true
+          input: true
+          url_param_only: true
+          pattern: projects/{{project}}/locations/global/certificateMaps/{{name}}
+          description: |
+            A user-defined name of the Certificate Map Entry. Certificate Map Entry 
+            names must be unique globally and match pattern 
+            'projects/*/locations/*/certificateMaps/*/certificateMapEntries/*'
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'description'
+          description: |
+            A human-readable description of the resource.
```

Comment:
Same comment with the description here. I think it helps to know it can be paragraphs.